src/content/plugins/internal-plugins.md

```diff
@@ -313,14 +286,6 @@ Merges chunks until each chunk has the minimum size of `minChunkSize`.
 
 Adds chunk ids of chunks which are included in the chunk. This eliminates unnecessary chunk loads.
 
-### OccurenceOrderPlugin
```

Comment:
Removed in https://github.com/webpack/webpack/commit/a2af69056ae3a477f65f9d3dbdf327e2d3c6e215